src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -179,9 +183,29 @@ def _fill_ftp_publishing_url(webapp, resource_group_name, name, slot=None):
     return webapp
 
 
+def _format_linux_fx_version(custom_image_name):
+    fx_version = custom_image_name
+    fx_version_lower = fx_version.lower()
+    # handles case of only spaces
+    if custom_image_name.strip():
```

Comment:
Any reason we need to handle white spaces? Values from command arguments should have them stripped by the command parser. 
If we do need to handle, then you need to use stripped version to construct the `fx_version` below.